src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -118,10 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="SetColorSchemeParentCommandName" xml:space="preserve">
-    <value>Select color scheme</value>
+    <value>Apply color scheme</value>
   </data>
   <data name="NewTabParentCommandName" xml:space="preserve">
-    <value>New tab</value>
+    <value>Open a new tab</value>
   </data>
   <data name="SplitPaneParentCommandName" xml:space="preserve">
     <value>Split pane</value>

```

Comment:
wait wasn't the whole point of #17684 that "Split pane" was ambiguous between the nested default action and the unnested one? Like, the whole point was to add back "Split pane<b>...</b>"

Reply:
OMG GOOD CATCH. This is what I get for leaving on vacation for a few days haha.

So far we've done...
❌ `Split pane`: collision with regular `splitPane` action which hides the nested version
❌ `Split pane...`:  maintains muscle memory, but `...` was considered wrong based on feedback in #16846 
❌ `Split a new pane`: maintains muscle memory, but `new` was considered wrong based on feedback in [this thread](https://github.com/microsoft/terminal/pull/17715#discussion_r1718475395).

Some other ideas I have:
❓`Split current pane`: personally don't like the similarity to plain `Split pane`
❓`Split pane using a profile`: This looks different enough and implies some ambiguity. Trying this one out.

Leaving thread unresolved to maintain a space for discussion.

CC @DHowett @lhecker since y'all were on the earlier thread.

Reply:
Can we just accept being slightly wrong (and slightly out of convention with the style guide) and bring back the `...`s on otherwise completely unmodified strings? :)

Reply:
Changed back to `Split pane...`